cmd/cosign/cli/copy/copy.go

```diff
@@ -71,6 +72,43 @@ func CopyCmd(ctx context.Context, regOpts options.RegistryOptions, srcImg, dstIm
 		return err
 	}
 
+	idx, isIndex := root.(oci.SignedImageIndex)
+
```

Comment:
I know a lot of this is copy-paste from `sbom download`. Is it possible to factor out?